PROTO_tests/tests/import_export_test.py

```diff
@@ -0,0 +1,135 @@
+import glob
+import os
+from shutil import rmtree
+import tempfile
+
+import numpy as np
+import pandas as pd
+import arkouda as ak
+import pytest
+from arkouda import io_util, io
+
+
+@pytest.fixture
+def ie_test_base_tmp(request):
+    ie_test_base_tmp = "{}/import_export_test".format(os.getcwd())
+    io_util.get_directory(ie_test_base_tmp)
+
+    # Define a finalizer function for teardown
+    def finalizer():
+        # Clean up any resources if needed
+        io_util.delete_directory(ie_test_base_tmp)
+
+    # Register the finalizer to ensure cleanup
+    request.addfinalizer(finalizer)
+    return ie_test_base_tmp
+
+
+class TestImportExport:
```

Comment:
I thought these were covered in `io_test` (we tried to refactor to have all the io tests in the same file using class names to differentiate them)

Reply:
Removed :)